guide/payments/introduction.md

```diff
@@ -46,7 +46,7 @@ An introduction to how receiving and requesting bitcoin works.
 
 ---
 
-### Transaction privacy (coming soon)
+### [Transaction privacy]({{ '/guide/payments/privacy/' | relative_url }})
 
 An overview of how to help users maintain their financial privacy while using Bitcoin.
```

Comment:
This reads more like a general privacy overview wheras this page mostly talks about privacy in the context of transactions.

Reply:
How about this @Bosch-0 ?

"An overview of how to help users maintain their financial privacy while transacting with Bitcoin."

Reply:
'An overview of how to help users maintain transaction privacy with Bitcoin' ?
This commit suggestion is anyway dealing with the missing link, so go ahead and commit.